pyproject.toml

```diff
@@ -22,7 +22,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.10"
-ojmicroline-thermostat = { git = "https://github.com/robbinjanssen/python-ojmicroline-thermostat.git", branch = "main" }
+ojmicroline-thermostat = { git = "https://github.com/adamjernst/python-ojmicroline-thermostat.git", branch = "main" }
```

Comment:
This change needs to be reverted when robbinjanssen/python-ojmicroline-thermostat#200 is merged; for now, it allows for testing.

Reply:
Has been released as 2.0.0 :-)

I'll test this asap

Reply:
@robbinjanssen Great! Please do test OWD5 flow especially since of course I cannot test that — both migrating an existing config, and setting up a new OWD5 config. I will test UWG4 thoroughly.

There are obviously rough edges in this PR that need to be finished off, I'll work on it soon!